packages-exp/performance-exp/package.json

```diff
@@ -12,7 +12,7 @@
   "scripts": {
     "lint": "eslint -c .eslintrc.js '**/*.ts' --ignore-path '../../.gitignore'",
     "lint:fix": "eslint --fix -c .eslintrc.js '**/*.ts'",
-    "build": "rollup -c",
+    "build": "rollup -c && yarn api-report",
```

Comment:
@jposuna 
Added api-report as part of the build process. It will update the API review file if any API changed.

Reply:
Great, thank you!